### ISSUE TITLE
tensorrt

### ISSUE THREAD

**Issue by liangjunhao1998** [CoinCheung/BiSeNet#294]
![image](https://user-images.githubusercontent.com/85820796/222034739-b96cc15d-cc7f-4a06-ba58-060fd06ae3fc.png)
想问一下这图里的代码是什么，作用是什么

**Comment by liangjunhao1998** [comment]
max_val 代表什么意思，我看输出是大于1的，语义评分？

**Comment by CoinCheung** [comment]
Hi,

sorry for being so late. It means max value of pixel logits(not softmax, so you see it is greater than 1).

**Comment by CoinCheung** [comment]
I am closing this since this question is answered.
